<?xml version="1.0"?>
<robot
    xmlns:xacro="http://www.ros.org/wiki/xacro" name="delta_common" >

  <m_f_rate value="-0.0146" /> <!-- drug torque rate -->
  <xacro:property name="max_force" value="28.2" /> <!-- [N] -->
  <xacro:property name="min_force" value="0.0" /> <!-- [N] -->

  <rotor_tilt1 value="-0.125"/>
  <rotor_tilt2  value="0.146"/>
  <rotor_tilt3  value="0.083"/>
  <xacro:property name="rotor_tilt1" value="-0.125"/>
  <xacro:property name="rotor_tilt2" value="0.146"/>
  <xacro:property name="rotor_tilt3" value="0.083"/>

</robot>
